src/utils/metrics.ts

```diff
@@ -169,6 +169,13 @@ export function createMetrics(registry: Registry, register = true) {
         registers
     })
 
+    const emittedEvents = new Counter({
+        name: "emitted_events",
```

Comment:
```suggestion
        name: "alto_emitted_user_operation_events",
```

1) `alto_` prefix is necessary
2) added `user_operation` for more clarity